pandas/tests/io/conftest.py

```diff
@@ -37,6 +37,8 @@ def s3_resource(tips_file, jsonl_file):
     """
     pytest.importorskip('s3fs')
     boto3 = pytest.importorskip('boto3')
+    # GH-24092. See if boto.plugin skips the test or fails.
+    pytest.importorskip("boto.plugin")
```

Comment:
For some reason, this line fails instead of skipping. https://travis-ci.org/pandas-dev/pandas/jobs/463496968

That job is using the newest `pytest 4.0.1`, but maybe there's a bug in `importorskip` for submodules?

Reply:
Going to try to avoid pinning pytest for now.